tom-fitz/figure-assessment
# Development

**Test**
- make sure you have a local cluster running with a test env already loaded.
- Here is a test deployment:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-database-deployment
spec:
  replicas: 1
  selector:
    matchLabels:
      app: test-database
  template:
    metadata:
      labels:
        app: test-database
    spec:
      containers:
      - name: nginx
        image: nginx:latest
```
- run the script - `cd /infra/scripts/ && go run ./db_redeployment.go`
- add print lines to show pods before and after for more verbose testing
